backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxBackend.scala

```diff
@@ -272,6 +272,10 @@ object VeloxBackendSettings extends BackendSettingsApi {
 
   override def supportNativeMetadataColumns(): Boolean = true
 
+  override def supportNativeRowIndexColumn(): Boolean = {
+    GlutenConfig.getConf.enableNativeRowIndexColumn
+  }
```

Comment:
Can't we use `true` instead of a config here? similar as `supportNativeMetadataColumns`.

Reply:
I will update this to always `true` when this velox fix ready https://github.com/facebookincubator/velox/pull/9944